images/repo-brancher-controller/Dockerfile

```
@@ -0,0 +1,4 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
+
+ADD repo-brancher-controller /usr/bin/repo-brancher-controller
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)